cypress/support/step-definitions/menu-steps.js

```diff
@@ -21,3 +22,11 @@ Then('Menu third expandable element has inner elements', () => {
   innerMenu(positionOfElement('fourth')).should('have.attr', 'data-component', 'link');
   innerMenu(positionOfElement('fifth')).should('have.attr', 'data-component', 'link');
 });
+
+When('I open {int}', (index) => {
+  submenu().eq(index - 1).trigger('mouseover');
```

Comment:
we could use here the function `positionOfElement()` and pass here a `{string}` as `first`/`second` instead of passing `{int}`  and subtracting some `magic numbers`.